docs/dev/plugins.rst

```diff
@@ -238,6 +238,15 @@ The events currently available are:
   during a ``beet import`` interactive session. Plugins can use this event for
   :ref:`appending choices to the prompt <append_prompt_choices>` by returning a
   list of ``PromptChoices``. Parameters: ``task`` and ``session``.
+  
+* `mb_track_extract`: called after the metadata is obtained from
+  MusicBrainz. The parameter is a ``dict`` containing the data
+  retrieved from MusicBrainz for a track.
+  Allows to add additional tags.
+  Parameter: ``data``
+
+* `mb_album_extract`: Like `mb_track_extract`, but for album tags.
```

Comment:
One more nitpick here: I guess it would be helpful if it was spelled out how exactly plugins can add these additional tags. Similar to how the documentation for `before_choose_candidate` above says what type plugins need to return, the return type could be named here (a dict with the `field: value` mapping I suppose?).

Reply:
Done, does it correspond to what you have in mind?

Reply:
I added it, I hope it's clear and understandable. I also added precisions about priorities (i. e. who prevails when the field is the same).